src/KeyVault/KeyVault/help/Import-AzKeyVaultSecurityDomain.md

```diff
@@ -26,6 +26,13 @@ Import-AzKeyVaultSecurityDomain -Keys <KeyPath[]> -SecurityDomainPath <String> [
  [-SubscriptionId <String>] [<CommonParameters>]
 ```
 
+### ByRestoreBlob
+```
+Import-AzKeyVaultSecurityDomain -SecurityDomainPath <String> [-RestoreBlob] [-PassThru]
```

Comment:
RestoreBlob is an unnecessary parameter. If keys is not bounded, we can assume its a decrypted security domain